interface/super/edit_layout.php

```diff
@@ -1271,21 +1281,41 @@ function myChangeCheck() {
 ?>
 </select>
 </span>
-<div><p>
 <?php if ($layout_id) { ?>
 <input type='button' value='<?php echo xla('Layout Properties'); ?>' onclick='edit_layout_props("")' />&nbsp;
 <input type='button' class='addgroup'  id='addgroup'  value='<?php echo xla('Add Group'); ?>' />
 <span style="font-size:90%"> &nbsp;
 <input type='button' class="btn btn-danger"name='save' id='save' value='<?php xl('Save Changes', 'e'); ?>' /></span> &nbsp;&nbsp;
 <?php xl('With selected:', 'e');?>
 <input type='button' name='deletefields' id='deletefields' value='<?php xl('Delete', 'e'); ?>' style="font-size:90%" disabled="disabled" />
-<input type='button' name='movefields' id='movefields' value='<?php xl('Move to...', 'e'); ?>' style="font-size:90%" disabled="disabled" /></span>
+<input type='button' name='movefields' id='movefields' value='<?php xl('Move to...', 'e'); ?>' style="font-size:90%" disabled="disabled" />
+<input type='button' value='<?php echo xla('Tips'); ?>' onclick='$("#tips").toggle();' />&nbsp;
+</span>
 <?php } else { ?>
 <input type='button' value='<?php echo xla('New Layout'); ?>' onclick='edit_layout_props("")' />&nbsp;
 <?php } ?>
-</p></div>
-</div>
 
+<div id="tips" class="container tips"><section class="panel panel-primary">
+  <header class="panel-heading">
+   <h3 class="panel-title">Usage Tips</h3>
```

Comment:
After substantial digging here, I have finally found something that needs to be translated :)

Reply:
Honestly my friend, I truly do not have anything against translating for our non English speaking brothers and sisters:) I am however a little put off we don't translate southerner!